lib/events/codes.go

```diff
@@ -620,6 +620,13 @@ const (
 	// AccessGraphAccessPathChangedCode is the access graph access path changed event code.
 	AccessGraphAccessPathChangedCode = "TAG001I"
 
+	// IntegrationCreateCode is the integration resource create event code.
+	IntegrationCreateCode = "IG001"
+	// IntegrationUpdateCode is the integration resource update event code.
+	IntegrationUpdateCode = "IG002"
+	// IntegrationDeleteCode is the integartion resource delete event code.
+	IntegrationDeleteCode = "IG003"
```

Comment:
Nit: Can you add `I` suffix (indicates "Info") to codes to be consistent with other events?

```suggestion
	// IntegrationCreateCode is the integration resource create event code.
	IntegrationCreateCode = "IG001I"
	// IntegrationUpdateCode is the integration resource update event code.
	IntegrationUpdateCode = "IG002I"
	// IntegrationDeleteCode is the integration resource delete event code.
	IntegrationDeleteCode = "IG003I"
```

Reply:
Ah forgot that, thanks